client/src/main/java/io/pravega/client/stream/ReaderGroupConfig.java

```diff
@@ -49,11 +49,41 @@
 
     private final int maxOutstandingCheckpointRequest;
 
+    @Getter
+    private final boolean isSubscriber;
+    @Getter
+    private final boolean autoTruncateAtLastCheckpoint;
```

Comment:
in same breadth as whatever we decide to rename the api to, we should rename this property too .. perhaps auto-publish-last-checkpoint?

Reply:
The name changes have been made please have a look.